db/migrate/20200331150436_rename_foreman_features.rb

```diff
@@ -0,0 +1,56 @@
+class RenameForemanFeatures < ActiveRecord::Migration[5.1]
+  class MiqProductFeature < ActiveRecord::Base; end
+  class MiqRolesFeature < ActiveRecord::Base; end
+
+  def up
+    MiqProductFeature.where(:identifier => 'provider_foreman_explorer').update(:identifier => 'ems_configuration')
+
+    MiqProductFeature.where(:identifier => 'provider_foreman_view').update(
+      :name        => 'View',
+      :description => 'View Configuration Manager Providers',
```

Comment:
For simplicity I think you can leave out the name and description from the migration, as the seed with fill out the rest of the details (unless of course you need it because of other uniqueness contraints).  The identifier is really the only thing that matters from a seeding perspective.